server/src/main/java/org/opensearch/search/pipeline/Pipeline.java

```diff
@@ -132,7 +137,11 @@ SearchRequest transformRequest(SearchRequest request) throws SearchPipelineProce
                         request = processor.processRequest(request);
                     } catch (Exception e) {
                         onRequestProcessorFailed(processor);
-                        throw e;
+                        if (processor.isIgnoreFailure()) {
+                            logger.warn("Failed to process request: " + processor.getType() + ". An error occurred in the processor.", e);
```

Comment:
This is not a great error message, since looks like the processor type is the error.

I would go with something like:

```
logger.warn("Search pipeline [" + id + "] exception from request processor [" + processor.getType() + "] ignored", e);
```

It's a single sentence that communicates which pipeline, which type of processor had the error, and mentions that the exception was ignored.

Reply:
changed